tests/e2e/scale/noobaa/test_delete_objects.py

```diff
@@ -0,0 +1,111 @@
+import logging
+import pytest
+
+from datetime import datetime
+from ocs_ci.ocs import hsbench
+from ocs_ci.ocs.bucket_utils import (
+    s3_delete_object,
+    s3_delete_objects,
+)
+from ocs_ci.framework.pytest_customization.marks import bugzilla, polarion_id
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="class")
+def s3bench(request):
+
+    s3bench = hsbench.HsBench()
+    s3bench.create_resource_hsbench()
+    s3bench.install_hsbench()
+
+    def teardown():
+        s3bench.cleanup()
+
+    request.addfinalizer(teardown)
+    return s3bench
+
+
+class TestDeleteObjects:
+    @bugzilla("2181535")
+    @polarion_id("OCS-4916")
+    @pytest.mark.parametrize(
+        argnames=["delete_mode"],
+        argvalues=[
+            pytest.param("single"),
+            # pytest.param("batch"),
+            # pytest.param("whole"),
+        ],
+    )
+    def test_delete_objects(
+        self,
+        scale_noobaa_db_pod_pv_size,
+        scale_noobaa_pods_resources,
+        bucket_factory,
+        s3bench,
+        mcg_obj,
+        delete_mode,
+    ):
+        """
+        Test deletion of objects, objectbucket and backingstore when there
+        is huge number of objects (~2 million) stored
+        """
+        scale_noobaa_db_pod_pv_size(pv_size="600")
+
+        bucket_class_dict = {
+            "interface": "OC",
+            "backingstore_dict": {"aws": [(1, "eu-central-1")]},
+        }
+        # create an object bucket
+        bucket = bucket_factory(bucketclass=bucket_class_dict, verify_health=False)[0]
+        bucket.verify_health(timeout=300)
+
+        # write 4K and 24M size objects of 1M each to the bucket
```

Comment:
this needs to be 4K and 4M instead of 24M?

Reply:
correct